tests/post/steps/test_salt_metalk8s_kubernetes_cronjob.py

```diff
@@ -0,0 +1,287 @@
+import json
+import os
+import time
+
+from kubernetes.client.rest import ApiException
+import pytest
+from pytest_bdd import parsers, scenario, given, then, when
+import yaml
+
+from tests.utils import negation, run_salt_command
+
+
+# Fixtures {{{
+
+
+@pytest.fixture(scope="function")
+def context():
+    return {}
+
+
+# }}}
+# Scenario {{{
+
+
+@scenario(
+    "../features/salt_metalk8s_kubernetes_cronjob.feature",
+    "Ensure the test CronJob is present",
+)
+def test_ensure_the_test_cronjob_is_present(host):
+    pass
+
+
+@scenario(
+    "../features/salt_metalk8s_kubernetes_cronjob.feature",
+    "Suspend and activate a CronJob",
+)
+def test_suspend_cronjob(host):
+    pass
+
+
+@scenario(
+    "../features/salt_metalk8s_kubernetes_cronjob.feature",
+    "Stop the Cronjob and delete all its Jobs",
+)
+def test_suspend_cronjob_delete_jobs(host):
+    pass
+
+
+# }}}
+# Given {{{
+
+
+@given(
+    parsers.parse("the '{cronjob_name}' CronJob is created"),
+)
+def call_metalk8s_kubernetes_cronjob_create_test_cronjob(
+    k8s_client, cronjob_name, utils_image
+):
+    cj_manifest_file = os.path.join(
+        os.path.realpath(os.path.dirname(__file__)), "files", "cronjob.yaml"
+    )
+    with open(cj_manifest_file, encoding="utf-8") as fd:
+        cj_manifest = yaml.safe_load(fd)
+    cj_manifest["metadata"] = {"name": cronjob_name}
+    cj_manifest["spec"]["jobTemplate"]["spec"]["template"]["spec"]["containers"][0][
+        "image"
+    ] = utils_image
+    cj_k8s_client = k8s_client.resources.get(api_version="batch/v1", kind="CronJob")
+    try:
+        cj_k8s_client.delete(name=cronjob_name, namespace="default")
+    except ApiException as exc:
+        if exc.status != 404:
+            raise exc
+    cj_k8s_client.create(body=cj_manifest, namespace="default")
+
+    yield True
```

Comment:
nit: Just `yield` is enough here

```suggestion
    yield
```

Reply:
Changed.